src/components/CallToAction.tsx

```diff
@@ -0,0 +1,51 @@
+import React from 'react';
+import { css } from 'emotion';
+import { textSans } from '@guardian/src-foundations/typography';
+import { palette } from '@guardian/src-foundations';
+
+const link = css`
+    background: ${palette.brandYellow.main};
+    border-radius: 21px;
+    box-sizing: border-box;
+    color: ${palette.neutral[7]};
+    ${textSans.medium({ fontWeight: 'bold' })};
+    text-decoration: none;
+    padding: 12px 46px 12px 21px;
+    line-height: 18px;
+    position: relative;
+    margin: 4px 10px 4px 0;
+    transition: background-color 0.3s ease-in-out;
+
+    :hover {
+        background: ${palette.brandYellow.dark};
+    }
+
+    svg {
+        fill: currentColor;
+        position: absolute;
+        right: 12px;
+        top: 50%;
+        height: 30px;
+        width: 30px;
+        transform: translate(0, -50%);
+        transition: transform 0.3s ease-in-out;
+    }
+
+    :hover svg {
+        transform: translate(3px, -50%);
+    }
+`;
+
+export const CallToAction: React.FC<{
+    url: string;
```

Comment:
Oh, what's the syntax here? I'm surprised the semicolons work 😄

Reply:
Chatted offline about this - I'd just gotten mixed up because I've been working with Flow in frontend lately (which would use commas in the type definition).

Reply:
Yup, as discussed IRL that's the syntax we need here